hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieDatasetBulkInsertHelper.scala

```diff
@@ -144,7 +149,7 @@ object HoodieDatasetBulkInsertHelper
                  arePartitionRecordsSorted: Boolean,
                  shouldPreserveHoodieMetadata: Boolean): HoodieData[WriteStatus] = {
     val schema = dataset.schema
-    val writeStatuses = dataset.queryExecution.toRdd.mapPartitions(iter => {
+    val writeStatuses = injectSQLConf(dataset.queryExecution.toRdd.mapPartitions(iter => {
```

Comment:
Could we have some test validations?

Reply:
Working on it, it's proving to be pretty difficult to make a test for this

Reply:
Our test data generator only produces timestamps that are after 1970 and it only occurs with deltastreamer, so there isn't existing infra to support a test like this

Reply:
ok, was able to get a test that will fail with master, but pass with this